knowledge-base/chart-tooltip-series-item.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Add a tooltip to the Chart series items
+description: An example on how you can add a tooltip for the Chart series items
+type: how-to
+page_title: Add a tooltip for the Chart series items - Kendo UI for Vue Native Chart
+slug: chart-tooltip-series-item
+tags: chart, series, tooltip
+res_type: kb
+category: knowledge-base
+---
+
+## Environment
+
+<table>
+    <tbody>
+	    <tr>
+	    	<td>Product Version</td>
+	    	<td>4.3.1</td>
+	    </tr>
+	    <tr>
+	    	<td>Product</td>
+	    	<td>Progress® Kendo UI for Vue Native</td>
+	    </tr>
+    </tbody>
+</table>
+
+
+## Description
+
+How to add a tooltip to the Chart series items?
```

Comment:
It was not clear to me that the tooltip will be shown only when I hover over the labels in the y axis. Can you update the description to state that, so the clients will not expect them over the actual series items of the chart.